src/content-helper/editor-sidebar/editor-sidebar.tsx

```diff
@@ -114,6 +131,18 @@ const getSettingsFromJson = ( settingsJson: string ): SidebarSettings => {
 	}
 
 	// Fix invalid values if any are found.
+	if ( typeof parsedSettings?.CrossLinksMaxLinkLength !== 'number' ) {
+		parsedSettings.CrossLinksMaxLinkLength = 4;
```

Comment:
I've noticed that we set the `4` and `10` values in multiple places. I'd try to minimize this if possible, but fine if this is needed. Alternatively, constants could be used for the values in order to avoid forgetting setting some of them.

Reply:
Re: the default value of `max_links`, I found it tricky since it can depend on the length of the text (number of paragraphs) selected?

Reply:
One idea would be to set this default value dynamically. Is it possible to say set the number of links to 5% or 10% of the total word count being selected?

Reply:
Maybe it could be mapped to a function instead of a constant. Let's see what @vaurdan says.